### DISCUSSION TITLE
add option in inpaint image processing only with SD 1.5

### DISCUSSION THREAD

**Discussion by elyzionz** [lllyasviel/Fooocus#991] [Ideas]
It would be great if in the inpaint option they could add the option to process the image only with SD1.5 and be able to configure the weight without having to use the SDXL model. That way the image processing would be much faster since it would not need the SDXL model. I tried the option to generate a new image using SDXL and SD 1.5 at the same time and the results are not as true to the message. offers better images and more faithful to the message using only SDXL, so for people who want an image generated only with SDXL and then process them only with SD 1.5 to create more realistic people without having to also use the SDXL model, since the one has a medium capacity GPU it takes longer, for example in automatic1111 when I create an SDXL image of a person I go to the impaint option and process that image only with a SD 1.5 model and the image processing is super fast and adds realism to the skin and face maintaining the fidelity of the message

image generated only with SDXL:

![2023-11-16_01-36-06_2077](https://github.com/lllyasviel/Fooocus/assets/23017459/718f2731-3845-4f24-980c-0a8812307f7a)

Image processed in inpaint with only SD 1.5 in automatic1111:

![00001-3614370689](https://github.com/lllyasviel/Fooocus/assets/23017459/c0326271-c7f4-4039-babd-12f233baccc2)

image generated with sdxl and sd1.5 at the same time in fooocus:

![2023-11-16_01-33-43_4901](https://github.com/lllyasviel/Fooocus/assets/23017459/9a14ed3c-92d8-4d00-a812-2a4c8fbd6ea1)

As you can see, it is better to generate the image first only with SDXL to make it more faithful to the prompt and then process it only with SD1.5 to give realism to the person's skin. To do this I have to use fooocus to generate the image and then use automatic1111 to give it realism, the idea would be to be able to do both in fooocus

**Comment by lllyasviel** [comment]
duplicated with https://github.com/lllyasviel/Fooocus/discussions/788
